project/src/main/java/com/google/sps/database/ConnectionPoolContextListener.java

```diff
@@ -133,7 +134,8 @@ public void contextInitialized(ServletContextEvent event) {
     }
   }
 
-  public void insertTestData(DataSource pool) throws SQLException {
+  //TODO(nemleon): deserves to be factored out
```

Comment:
I'm not too sure what this means, are we leaving a todo in the code or has it been sorted?

Reply:
I added a todo. This deserves to be taken out, but I have stopped short of doing it